.travis.yml

```diff
@@ -7,6 +7,7 @@ sudo: false
 php:
     - 7.2
```

Comment:
If in composer.json     "php": "^7.3", "sylius/sylius": "~1.7"  then remove 7.2 here
Or allow for both versions of sylius in composer.json ( https://bitbag.slack.com/archives/C6T41372S/p1591263679026500?thread_ts=1591263679.026500 )